### ISSUE TITLE
AssociateAddress Unauthorized Operation

### ISSUE THREAD

**Issue by mstratford** [aws-quickstart/quickstart-linux-bastion#133]
Hi there,

Since #129, my bastion quickstart launches seem to be failing (b-boostrap fails). I'm using commit 297feeb5aefd6d4826367ea7568a55de9e3c70c0

This is a Amazon-Linux2-HVM run.

The IP X.X.X.X is the IP address of the elastic IP the template created, but it was not able to associate, as such the bastion instance is using a standard Public IP from the subnet.

Are we missing a permission in the quickstart, or is there some odd behaviour where it's trying to associate the incorrect IP?

From `/var/log/cfn-init-cmd.log`, this was the first run during quickstart launch.
```
2021-07-01 17:19:18,373 P11410 [INFO]   Querying the assigned public IP
2021-07-01 17:19:18,373 P11410 [INFO]     % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
2021-07-01 17:19:18,373 P11410 [INFO]                                    Dload  Upload   Total   Spent    Left  Speed
2021-07-01 17:19:18,373 P11410 [INFO]   
2021-07-01 17:19:18,373 P11410 [INFO]     0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
2021-07-01 17:19:18,373 P11410 [INFO]   100    56  100    56    0     0  14000      0 --:--:-- --:--:-- --:--:-- 14000
2021-07-01 17:19:18,373 P11410 [INFO]   Determining EIP Association Status for [<?xml]
2021-07-01 17:19:18,373 P11410 [INFO]   Determining EIP Association Status for [X.X.X.X]
2021-07-01 17:19:18,373 P11410 [INFO]   Determining EIP Allocation for [X.X.X.X]
2021-07-01 17:19:18,373 P11410 [INFO]   
2021-07-01 17:19:18,373 P11410 [INFO]   An error occurred (UnauthorizedOperation) when calling the AssociateAddress operation: You are not authorized to perform this operation. Encoded authorization failure message: XXXXXXXXXX
2021-07-01 17:19:18,373 P11410 [INFO]   Unable to associate EIP X.X.X.X. Failure. Exiting
2021-07-01 17:19:18,373 P11410 [INFO] ------------------------------------------------------------
```

Cheers!

**Comment by mstratford** [comment]
@andrew-glenn Was there a particular reason / fix which caused #129 to be discovered / fixed? Maybe https://github.com/aws-quickstart/quickstart-linux-bastion/commit/b18c4c33ade637ad4366f33300c0e2df5cfe54bf is too restrictive?

**Comment by mstratford** [comment]
This was seemingly fixed in a later commit.
